README.md

```diff
@@ -264,6 +267,37 @@ Each organization/tenant is associated to a different database.
 
 [Top](#top)
 
+###<a name="section3.4"></a>OrionMongoSink persistence
+Then `OrionMongoSink` will persist the data within the body as:
+
+    $ mongo
+    MongoDB shell version: 2.6.9
+    connecting to: test
+    > show databases
+    admin              (empty)
+    local              0.031GB
+    vehicles           0.031GB
+    test               0.031GB
+    > use vehicles
+    switched to db vehicles
+    > show collections
+    4wheels
+    4wheels_car1_car
+    4wheels_car1_car_speed
+    system.indexes
+    > db.4wheels.find()
+    { "_id" : ObjectId("5534d143fa701f0be751db82"), "recvTimeTs": "1402409899391", "recvTime" : "2015-04-20T12:13:22.41.124UTC", "entityId" : "car1", "entityType" : "car", "attrName" : "speed", "attrType" : "kmh", "attrValue" : "112.9" }
```

Comment:
@frbattid have been discussing offline about using Date() for `recvTimeTs` so time based queries can be easily implemented. However, this would be deferred to a tech debt issue.